OpenSim/Simulation/Model/PrescribedForce.cpp

```diff
@@ -37,19 +37,24 @@ using namespace std;
 //=============================================================================
 // CONSTRUCTOR(S) AND DESTRUCTOR
 //=============================================================================
-// default destructor, copy constructor, copy assignment
+// default destructor
 //_____________________________________________________________________________
+PrescribedForce::PrescribedForce()
+{
+    setNull();
+    constructInfrastructure();
+
+}
+
 /**
- * Default constructor.
+ * Convenience constructor.
  */
-PrescribedForce::PrescribedForce(Body* body)
+PrescribedForce::PrescribedForce(const PhysicalFrame& frame)
 {
```

Comment:
It would be great if the convenience constructor required the Force name as well. This helps us avoid having to rename retroactively.